packages/ts/react-crud/src/autogrid-renderers.tsx

```diff
@@ -70,3 +72,37 @@ export function AutoGridDateTimeRenderer<TItem>({ item }: RendererOptions<TItem>
 export function AutoGridRowNumberRenderer<TItem>({ model }: RendererOptions<TItem>): JSX.Element {
   return <>{model.index + 1}</>;
 }
+
+export function AutoGridFooterItemCountRenderer(
+  itemCountHolder: AutoGridItemCountHolder,
+  columns: JSX.Element[],
+  footerCountRenderer?: ComponentType<{ itemCountHolder: AutoGridItemCountHolder }>,
+): JSX.Element[] {
+  let newColumns: JSX.Element[] = columns;
```

Comment:
Maybe nitpicking (feel free to ignore):
Could the `newColumns` variable be omitted completely, and the if-else be simplified as:
```tsx
if (footerCountRenderer) {
    const FooterRenderer = footerCountRenderer;
    return [
      <GridColumnGroup key="grid-footer" footerRenderer={() => <FooterRenderer itemCountHolder={itemCountHolder} />}>
        {columns}
      </GridColumnGroup>,
    ];
}
// the else logic goes here without an explicit else block
```